### ISSUE TITLE
Incorrect UML diagram

### ISSUE THREAD

**Issue by TimothyLKM** [TimothyLKM/pe#11]
![Screenshot 2024-04-19 at 5.44.14 PM.png](https://raw.githubusercontent.com/TimothyLKM/pe/main/files/e6dd8f5a-ecd0-412e-9450-14c8728fa0d2.png)

If Argument List is not being used after "return parsedArguments", the rectangle for :ArgumentList do not need to stretch to the end. It can end off after parsedArguments is return.

**Comment by nus-se-script** [comment]
# Team's Response
Thank you for pointing this out. It is indeed an issue that the activation bar should have been closed at `return parsedArguments`. This issue is a duplicate of #919 as they both raised the same issue with the activation bar closing too late and should have been closed earlier at `return parsedArguments`.

## The 'Original' Bug

[The team marked this bug as a duplicate of the following bug]

># Usage of a class is not closed even after finish using it
>
>**Note from the teaching team:** This bug was reported during the _Part II (Evaluating Documents)_ stage of the PE. **You may reject this bug if it is not related to the quality of documentation.**<hr>
>![image.png](https://raw.githubusercontent.com/nus-cs2113-AY2324S2/pe/master/files/981ebbff-636f-4fea-add7-04917f431b31.png)
>
>Here argument list class being opened at the start but its not being closed even after finished using it, its only closed at the end. Consider closing it once it is finished using just like ModifyCommand.
>
>
>
>
>
>
><hr>
><sub>[original: nus-cs2113-AY2324S2/pe-interim#841] [original labels: type.DocumentationBug severity.VeryLow]</sub>
>
>
>

## Their Response to the 'Original' Bug

[This is the team's response to the above 'original' bug]

>Thank you for pointing this out. It is indeed an issue that the activation bar should have been closed at `return parsedArguments`.
# Items for the Tester to Verify
## :question: Issue duplicate status

Team chose to mark this issue as a duplicate of another issue (as explained in the _**Team's response**_ above)

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

<catcher-end-of-segment><hr>
## :question: Issue severity

Team chose [`severity.VeryLow`]
Originally [`severity.Low`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

<catcher-end-of-segment><hr>
